Sofa/framework/Core/src/sofa/core/behavior/LinearSolver.h

```diff
@@ -74,6 +74,12 @@ class SOFA_CORE_API LinearSolver : public BaseLinearSolver
     /// Solve the system as constructed using the previous methods
     virtual void solveSystem() = 0;
 
+    SOFA_ATTRIBUTE_DISABLED("v22.06", "v22.12", "LinearSolver does not offer any partial_solve anymore. Please refer to core::behavior::PartialLinearSolver if you wish to use partial solving features.")
```

Comment:
Should you follow the same principle than in https://github.com/sofa-framework/sofa/pull/3236 ?

Reply:
A bit different case:  #3226 was generating warnings because the deprecated members were used/constructed by the component constructor.
Here, it is deleted functions so there are not used (so they dont throw warnings)

Reply:
gotcha